config/manifest.yaml

```diff
@@ -0,0 +1 @@
+DOCKER_FILE_PATH: helm-init-container/Dockerfile
```

Comment:
Can you add the helm chart key as well with value <None>, just want to see if the pipeline skips the chart creation